lib/stackConfiguration/src/main/resources/stack.properties

```diff
@@ -311,6 +311,48 @@ qZSZQ/z3KBXZMZvjC2ac5hF+NtHZzLn3Vm0ltd9VAoGBAJcs9PBjJ9XqAHo6x88g\
 MjjcxwFQoYuOR/cgYAFyXY/jMXb6kKSdc02437hTXjTotvkgAJfO7gxVy8s3ipI7\
 pzvMOgD+6P3bCeoBuyKg2xhY
 
+# RSA is the standard for OpenID Connect: https://openid.net/specs/openid-connect-core-1_0.html#ServerMTI
+#
+# The following was geneated by:
+# openssl genpkey -aes256 -algorithm RSA -pkeyopt rsa_keygen_bits:2048 -out private-key.pem
+# openssl pkey -in private-key.pem -text
+# The keys in this file are for development/testing only and should never be used in production.
+#
+# Generally the following is a comma-separated list of pem-encoded private keys
+org.sagebionetworks.cloudfront.private.key.secret=\
+MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDcL+FZG6YvUVFV\
```

Comment:
This should not be actual key of dev or any environment. it should be a key only for build